src/main/java/com/bunq/sdk/http/BunqBasicHeader.java

```diff
@@ -1,26 +1,47 @@
 package com.bunq.sdk.http;
 
+import okhttp3.Response;
+
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
 public class BunqBasicHeader {
-  private String name;
-  private String value;
+  private static final String DELIMITER_HEADER_NAME_AND_VALUE = ": ";
```

Comment:
Missing constant group doc block. 

In this case it can be 

```
/**
* String format constants. 
*/
```

and perpend these 2 constants with `FORMAT_`